charts/misc-apps/values.yaml

```diff
@@ -73,7 +73,7 @@ metallb:
   # metallb.chart -- Chart
   chart: "metallb"
   # metallb.targetRevision -- [metallb Helm chart](https://github.com/bitnami/charts/tree/master/bitnami/metallb)
-  targetRevision: "1.0.*"
+  targetRevision: "2.0.*"
```

Comment:
Upgrade Notice:

https://github.com/bitnami/charts/tree/master/bitnami/metallb#to-200